### ISSUE TITLE
NBA LIVE 06 [EA-122]

### ISSUE THREAD

**Issue by gandalfthewhite19890404** [Cxbx-Reloaded/game-compatibility#831]
### Description

Booting, goes to the match but with broken graphics. Sound is missing.

### Dumps 

[Xbe_nba06.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/2134193/Xbe_nba06.txt)
[KrnlDebug_nba06.zip](https://github.com/Cxbx-Reloaded/game-compatibility/files/2134194/KrnlDebug_nba06.zip)

### Status
**Previous behavior:** 

None reported
**Current behavior:**

Broken graphics - almost nothing seen, no FMVs (blue screen instead of, just like in NBA Live 2005).

### Screenshots

![image](https://user-images.githubusercontent.com/24764335/41866523-172dd588-78b9-11e8-86c1-912d6053b7b2.png)
![image](https://user-images.githubusercontent.com/24764335/41866560-3075c17c-78b9-11e8-9e51-ec05932db956.png)

### Additional Information

Folder structure similar to NBA Live 2005, maybe both titles using similar engine and functions and it bring same issues.

**Comment by Voxel9** [comment]
When you say "goes to the match", does that mean it gets ingame, but with those broken graphics?

**Comment by gandalfthewhite19890404** [comment]
Yes, seems its a match because it responds to Start button - menu also broken, but if you press Start again match continues with same render issues as before press Start button.
